packages/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/GenerateCodegenArtifactsTaskTest.kt

```diff
@@ -145,7 +141,7 @@ class GenerateCodegenArtifactsTaskTest {
           writeText(
               """
         {
-            "name": "@a/library",
+            "name": "@a/libray",
```

Comment:
Was that intentional?

Reply:
That's a fix of a typo which got reverted in [ce9c8e0](https://github.com/facebook/react-native/pull/43049/commits/ce9c8e025c462e4ab0bdf9275ff7834805a70b1a).

I'll re-export this PR to be only the relevant part.

Reply:
@szymonrybczak now the new diff contains only the relevant changes